iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/DanglingLineAdderImpl.java

```diff
@@ -66,7 +66,7 @@ public GenerationAdder setTargetV(double targetV) {
         public DanglingLineAdder add() {
             ValidationUtil.checkActivePowerLimits(DanglingLineAdderImpl.this, minP, maxP);
             ValidationUtil.checkActivePowerSetpoint(DanglingLineAdderImpl.this, targetP);
-            ValidationUtil.checkVoltageControl(DanglingLineAdderImpl.this, voltageRegulationOn, targetV, targetQ);
+            ValidationUtil.checkVoltageControlTargetQ(DanglingLineAdderImpl.this, targetV, targetQ, voltageRegulationOn);
```

Comment:
Not sure why you changed the order of parameters (but it is not a blocking issue)